tasks/3/index.js

```diff
@@ -1,8 +1,14 @@
 // 3. From the array numbers to deduce a new array in which to replace
 // positive values on 1, negative values on -1, zero values leave
 
-const getReplacedArray = (array) => {
-  // your code
-};
+const getReplacedArray = (array) => array
+  .map(arr => {
```

Comment:
1. parameters of function write in parentheses
2. Replace the name of variable according to appointment
3. Use if...else if...else syntax